src/cascadia/TerminalConnection/AzureConnection.h

```diff
@@ -56,30 +53,30 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         void _RunStoreState();
         void _RunConnectState();
 
-        const utility::string_t _loginUri{ U("https://login.microsoftonline.com/") };
-        const utility::string_t _resourceUri{ U("https://management.azure.com/") };
-        const utility::string_t _wantedResource{ U("https://management.core.windows.net/") };
+        const std::wstring_view _loginUri{ L"https://login.microsoftonline.com/" };
+        const std::wstring_view _resourceUri{ L"https://management.azure.com/" };
+        const std::wstring_view _wantedResource{ L"https://management.core.windows.net/" };

```

Comment:
Why are these members and not `static constexpr`?

Reply:
I'll change it to (1) static constexpr for the default values but (2) local wstring_views that point at those -- because eventually _we will change all 3 of thee depending on which azure cloud you are using_